ts/store/reducers/backendStatus/remoteConfig.ts

```diff
@@ -296,6 +296,51 @@ export const isIdPayEnabledSelector = createSelector(
     )
 );
 
+export const absolutePortalLinksFallback = {
+  io_web: "https://ioapp.it/",
+  io_showcase: "https://io.italia.it/"
+};
+
+/**
+ * returns the absolute URLs for showcase and logout (io-web) sites
+ */
+export const absolutePortalLinksSelector = createSelector(
+  remoteConfigSelector,
+  remoteConfig =>
+    pipe(
+      remoteConfig,
+      O.map(config => config.absolutePortalLinks),
+      O.getOrElse(() => absolutePortalLinksFallback)
+    )
+);
+
+type hostType = keyof ReturnType<typeof absolutePortalLinksSelector>;
+
+/**
+ * Selector to generate a complete URL by replacing the base domain with a dynamic value from absolutePortalLinks.
+ * This is useful when URLs need to be dynamically constructed based on remote configuration or state.
+ */
+export const replaceBaseUrlSelector = createSelector(
```

Comment:
Actually here we don't replace anything, but we add the base url, am I right? What about changing the function name to something like `addBaseUrlSelector`. What about explicitly state that we add also the schema along with the base host? What do you think?